mpes/src/crawler.py

```diff
@@ -64,7 +64,7 @@ def crawl(year, month, output_path):
                 url = url_formats[key].format(year, sufix, month, url_code[after_2018_ot], months[int(month)])
         else:
             url = url_formats[key].format(year, month, url_code[after_2018_ot], months[int(month)])
-    
+        print(url)
```

Comment:
Isso vai dar problema quando a gente for executar o pipeline.

Reply:
Tava testando e esqueci de tirar